dags/transportation/dockless/v_trips.sql

```diff
@@ -0,0 +1,21 @@
+CREATE MATERIALIZED VIEW v_trips AS 
```

Comment:
Was this view missing before, or not checked in?

Reply:
not checked in.